src/plutosrc/dataAnalytics.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Park It</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;
+            background-color: #f4f4f4;
+            color: #333;
+        }
+        .container {
+            max-width: 600px;
+            margin: auto;
+            padding: 20px;
+            background-color: #fff;
+            border-radius: 8px;
+            box-shadow: 0 2px 4px rgba(0,0,0,0.1);
+            margin-top: 50px;
+        }
+        h1 {
+            color: #0066cc;
+            text-align: center;
+        }
+        #backButton {
+            display: block;
+            margin: 20px; /* Adjust top margin as needed */
+            padding: 10px 20px;
+            background-color: #0066cc;
+            color: #fff;
+            border: none;
+            border-radius: 4px;
+            cursor: pointer;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <h1>Park It!</h1>
+        <p>Data Analytics</p>
+        <button id="backButton">Return to Main Page</button>
```

Comment:
There should be only three buttons in a navigation bar, at the top of the website. A button for Live Data, one for Data Analytics and one for the about page. This navigation bar code should be copied and pasted into every html file and should look identical on all pages.